www/scripts/codecheckerviewer/CheckerStatistics.js

```diff
@@ -0,0 +1,204 @@
+// -------------------------------------------------------------------------
+//                     The CodeChecker Infrastructure
+//   This file is distributed under the University of Illinois Open Source
+//   License. See LICENSE.TXT for details.
+// -------------------------------------------------------------------------
+
+define([
+  'dojo/_base/declare',
+  'dojo/data/ItemFileWriteStore',
+  'dojo/Deferred',
+  'dojo/promise/all',
+  'dojo/store/Memory',
+  'dojo/store/Observable',
+  'dojox/form/CheckedMultiSelect',
+  'dojox/grid/DataGrid',
+  'dijit/layout/ContentPane',
+  'codechecker/hashHelper',
+  'codechecker/util'],
+function (declare, ItemFileWriteStore, Deferred, all, Memory, Observable,
+  CheckedMultiSelect, DataGrid, ContentPane, hashHelper, util) {
+  var RunFilter = declare(CheckedMultiSelect, {
+    _updateSelection: function() {
+      this.inherited(arguments);
+      if(this.dropDown && this.dropDownButton){
+        var label = '';
+
+        this.options.forEach(function(option) {
+          if (option.selected)
+            label += (label.length ? ', ' : '') + option.label;
+        });
+
+        this.dropDownButton.set('label', label.length ? label : this.label);
+      }
+    }
+  });
+
+  var FilterPane = declare(ContentPane, {
+    constructor : function () {
+      var that = this;
+
+      this._runStore = new Observable(new Memory({}));
+
+      this._runFilter = new RunFilter({
+        store: this._runStore,
+        labelAttr : 'label',
+        label : 'Get Statistics only for these runs...',
```

Comment:
`Get statistics only for runs...` would be better here. Also, if this is a user-string, perhaps  `"` should be used instead of `'`.

For the sake of readability, please align the `:`s under each other.